jib-gradle-plugin/README.md

```diff
@@ -250,15 +257,27 @@ jib {
 
 You can add arbitrary, non-classpath files to the image by placing them in a `src/main/jib` directory. This will copy all files within the `jib` folder to the image's root directory, maintaining the same structure (e.g. if you have a text file at `src/main/jib/dir/hello.txt`, then your image will contain `/dir/hello.txt` after being built with Jib).
 
-You can configure a different directory by using the `extraDirectory` parameter in your `build.gradle`:
+You can configure a different directory by using the `extraDirectory` parameter in your `build.gradle`, either by using the `jib.extraDirectory.path` parameter:
 
 ```groovy
-jib {
-  ...
-  // Copies files from 'src/main/custom-extra-dir' instead of 'src/main/jib'
-  extraDirectory = file('src/main/custom-extra-dir')
-  ...
-}
+// Copies files from 'src/main/custom-extra-dir' instead of 'src/main/jib'
+jib.extraDirectory.path = file('src/main/custom-extra-dir')
```

Comment:
Should we just encourage them to keep the same usage `jib.extraDirectory =` rather than showing this `.path` alternative, which may be confusing?